lib/sentry/config.ex

```diff
@@ -493,9 +493,10 @@ defmodule Sentry.Config do
                      hook_opts_schema
                    ])
 
-  @opts_schema NimbleOptions.new!(@raw_opts_schema)
   @valid_keys Keyword.keys(@raw_opts_schema)
 
+  defp opts_schema do: NimbleOptions.new!(@raw_opts_schema)
+
```

Comment:
@mayel this is a syntax error

Reply:
Oops sorry, did that quickly using github's editor. Can fix in a bit.

Reply:
done